gammapy/modeling/models/utils.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from astropy import units as u
+from astropy.io import fits
+from gammapy.maps import MapAxis, RegionNDMap
+from gammapy.utils.time import time_ref_from_dict
+
+
+def _read_cta_sdc(filename):
+    """To create a `LightCurveTemplateTemporalModel`
+    from the cta-sdc files. This format is subject to change"""
+    with fits.open(filename) as hdul:
+
+        energy_hdu = hdul["ENERGIES"]
+        energy_axis = MapAxis.from_nodes(
+            nodes=energy_hdu.data,
+            unit=energy_hdu.header["TUNIT1"],
+            name="energy",
+            interp="log",
+        )
+        time_hdu = hdul["TIMES"]
+        time_header = time_hdu.header
+        time_header.setdefault("MJDREFI", 0.5)
+        time_header.setdefault("MJDREFF", 55555)
```

Comment:
Should this be the opposite? I think that MJDREFI is integer part while MJDREFF is the float one.

Reply:
Indeed, my bad! Thanks @fabiopintore